Patch from upstream CVS, slightly modified

===================================================================
RCS file: /cvsroot/xchat/xchat2/src/fe-text/fe-text.c,v
retrieving revision 1.27
retrieving revision 1.28
diff -u -r1.27 -r1.28
--- xchat/xchat2/src/fe-text/fe-text.c	2005/11/04 03:39:23	1.27
+++ xchat/xchat2/src/fe-text/fe-text.c	2006/01/11 05:34:09	1.28
@@ -667,6 +667,14 @@
 {
 }
 void
+joind (int action, server *serv)
+{
+}
+void
+fe_userlist_set_selected (struct session *sess)
+{
+}
+void
 fe_dcc_add (struct DCC *dcc)
 {
 }
